translator/freecell_translator/functions.py

```diff
@@ -0,0 +1,116 @@
+from .constants import *
+
+import numpy as np
+
+# Convert given card to one hot encoded vector e.g. 7C = 01000000001000000
+# Every digit is stored as a separate value in the array
+def convert_card_to_one_hot_encoded_vector(card):
+    suit = CARD_SUITS[card[-1]]
+    value = CARD_VALUES[card[:-1]]
+    vector = list(map(int, list(suit + value)))
+    return vector
+
+def convert_board_to_array_of_one_hot_encoded_values(board):
+    # Fill entire array with no card symbol
+    array_one_hot_encoded = np.full((8, 19, 17), fill_value=list(map(int, list(NO_CARD))))
+    # Fill array with actual cards
+    for i, column in enumerate(board):
+        for j, card in enumerate(column):
+            array_one_hot_encoded[i][j] = convert_card_to_one_hot_encoded_vector(card)
+
+    return array_one_hot_encoded
+
+def convert_free_cells_to_array_of_one_hot_encoded_values(free_cell):
```

Comment:
zastanowiłbym się nad jakimiś krótszymi nazwami funkcji, bo tutaj to już trochę przesada

Reply:
Są krótsze nazwy